deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,124 @@ spec:
-    doing so).
+    ## Start using the operator
 
+    This operator deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion. For example, if you subscribe to this package
+    in namespace `my-namespace`, KubeFed will be deployed to manage objects _only_ in `my-namespace` in the host cluster and target clusters.
+    
+    The operator provides the following CRDs for instantiating the `KubeFed` in a given cluster:
+    * KubeFedWebHook
+    * KubeFed
+    
+    ### KubeFed deployment
+    In order to deploy kubefed, you need to follow these three steps:
+
+    * Install the operator in a single `Namespace` or in All `Namespaces` based on the operator `Installation Mode`.
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create a `KubeFedWebHook` resource to instatiate an admission webhook controller for kubefed. It's recommended to install this CR with `scope:Cluster`
+    to allow webhook controller to serve API requests coming from all `kubefed` control planes within the same cluster.
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFedWebHook
+    metadata:
+      name: kubefed-webhook-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    * Create a `KubeFed` resource to drive the installation of `KubeFed` setting the scope to be either `Namespaced` or `Cluster`. 
+    If you're planning to federate a cluster-scoped resource type like `StorageClass` then set the scope field to `Cluster`
+    
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFed
+    metadata:
+      name: kubefed-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    ```
+    $ kubectl get po -n <namespace> 
+    NAME                                          READY   STATUS    RESTARTS   AGE
+    kubefed-admission-webhook-77f7bd89dd-92kzj    1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-5t2fk   1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-b865z   1/1     Running   0          25s
+    kubefed-operator-db58d9b89-8694x              1/1     Running   0          61s
+    
+    ```
+    **Important Note**: 
+    * It's recommended to deploy kubefed in either a namespace-scoped or cluster-scoped mode but not both ways as deploying a cluster-scoped and 
+    namespace-scoped kubefed control plane to the same cluster will result in both control planes try to manage resources in the namespace of the namespace-scoped 
+    control plane and you will not see the desired outcome. For more information, use this [link](https://github.com/openshift/federation-dev/blob/master/docs/kubefed-scope.md) 
```

Comment:
s/and you will not see the desired outcome/and the managed resources will never converge on the desired state./

Reply:
Yeah, Thanks for filling this information for me. I know something was missing there.